### ISSUE TITLE
Is this actively maintained?

### ISSUE THREAD

**Issue by rtspeaks360** [crawlbase/crawlbase-python#1]
Creating an issue here to understand if this is actively maintained.

Please respond. I wanted to understand a few caveats around the use of available scrappers like Facebook via this wrapper.

@crawlbase

**Comment by crawlbase** [comment]
@rtspeaks360 yes this project is actively maintained. But please use this only to ask questions regarding this python project. If you have more generic questions about Crawlbase or our services, I suggest that you contact our [support team directly here](https://crawlbase.com/dashboard/support)
